docs/modules/ROOT/pages/how-tos/restore.adoc

```diff
@@ -230,3 +230,106 @@ dr-xr-xr-x  1 simonbeck  staff    0 Jun  8 09:21 tags
+   --runAsUser value                     Optional ; Set user UID, via cli or via env:  (default: 0) [$RUNASUSER]
+   --restoreToS3Endpoint value           Optional ; Set restore endpoint, only when using s3 restore method, via cli or via env: [$RESTORETOS3ENDPOINT]
+   --restoreToS3Bucket value             Optional ; Set restore bucket, only when using s3 restore method, via cli or via env: [$RESTORETOS3BUCKET]
+   --restoreToS3Secret value             Optional ; Set restore Secret, only when using s3 restore method, expecting secret name containing key value pair with 'username' and 'password' keys, via cli or via env: [$RESTORETOS3SECRET]
+   --RestoreToS3SecretUsernameKey value  Optional ; Set RestoreToS3SecretUsernameKey, key inside secret, under which S3 username is stored, via cli or via env:  (default: username) [$RESTORETOS3SECRETUSERNAMEKEY]
+   --RestoreToS3SecretPasswordKey value  Optional ; Set RestoreToS3SecretPasswordKey, key inside secret, under which Restic repo password is stored, via cli or via env:  (default: password) [$RESTORETOS3SECRETPASSWORDKEY]
+   --namespace value                     Optional ; Set namespace in which You want to execute restore, via cli or via env:  (default: default) [$NAMESPACE]
+   --kubeconfig value                    Optional ; Set kubeconfig to connect to cluster, via cli or via env: (default: ~/.kube/config) [$KUBECONFIG]
+   --help, -h                            show help (default: false)
+----
+
+Example usages:
+
+[source,bash]
+----
+# restore using PVC
+ k8up cli restore                                \
+  --restoreMethod pvc                            \
+  --kubeconfig .e2e-test/kind-kubeconfig-v1.24.4 \
+  --secretRef backup-repo                        \
+  --namespace default                            \
+  --s3endpoint http://minio:9000                 \
+  --s3bucket backups                             \
+  --s3secretRef minio-credentials                \
+  --snapshot f0dd5684                            \
+  --claimName wordpress-pvc                      \
+  --runAsUser 0
+
+
+# restore using S3 => S3
+ k8up cli restore                                \
+  --restoreMethod s3                             \
+  --kubeconfig .e2e-test/kind-kubeconfig-v1.24.4 \
+  --secretRef backup-repo                        \
+  --namespace default                            \
+  --s3endpoint http://minio:9000                 \
+  --s3bucket backups                             \
+  --s3secretRef minio-credentials                \
+  --snapshot f0dd5684                            \
+  --claimName wordpress-pvc                      \
+  --runAsUser 0                                  \
+  --restoreToS3Bucket backup2                    \
+  --restoreToS3Secret minio-credentials          \
+  --restoreToS3Endpoint http://minio:9000
+----
+
+It can be a little bit confusing to use all those flags, but it's pretty simple
+
+[source]
```

Comment:
I like this snipped, it really makes it simple to understand what flag is for what. 👍